posthog/models/integration.py

```diff
@@ -254,6 +265,23 @@ def oauth_config_for_kind(cls, kind: str) -> OauthConfig:
                 id_path="id",
                 name_path="email",
             )
+        elif kind == "linear":
+            if not settings.LINEAR_APP_CLIENT_ID or not settings.LINEAR_APP_CLIENT_SECRET:
+                raise NotImplementedError("Linear app not configured")
+
+            return OauthConfig(
+                authorize_url="https://linear.app/oauth/authorize",
+                additional_authorize_params={"actor": "application"},
+                token_url="https://api.linear.app/oauth/token",
+                token_info_url="https://api.linear.app/graphql",
+                token_info_graphql_query="{ viewer { organization { id name } } }",
+                token_info_config_fields=["data.viewer.organization.id", "data.viewer.organization.name"],
+                client_id=settings.LINEAR_APP_CLIENT_ID,
+                client_secret=settings.LINEAR_APP_CLIENT_SECRET,
```

Comment:
Can you give the CDP team + Tim access to the OAuth app and add a link to the oauth app to this page: https://github.com/PostHog/product-internal/issues/679